Source/ACE.Server/Program_DbUpdates.cs

```diff
@@ -191,7 +191,7 @@ private static void AutoApplyWorldCustomizations()
                 var contentDI = new DirectoryInfo(path);
                 if (contentDI.Exists)
                 {
-                    Console.Write($"Searching for SQL files within {path} .... ");
+                    Console.Write($"Searching for SQL files within {path} ....\n");
```

Comment:
Console.WriteLine would be the standard here

the EOL character might vary depending on OS environment